ecommerce/overview.rst

```diff
@@ -1,9 +0,0 @@
-=================
-Overview
-=================
```

Comment:
Removing this doc will likely raise warnings. Did you try building it?

Reply:
The issue here is that all RST files (doc pages) must be referenced by another RST file, in order to build the content tree of the documentation. In practice, `index.rst` references `ecommerce.rst` which itself references `overview.rst` and so on.
If you completely delete a doc page, the references to its RST file must be removed as well. If you make a global search (which whatever tool you like: PyCharm, Notepadd++, ...), you'll see that there is a line `overview.rst` in the `ecommerce.rst` file. It must be removed.

If you build (`make clean html`) the documentation on your branch, you'll see a warning for every 'dead' reference.